docs/src/main/asciidoc/_elytron/Credential_Store.adoc

```diff
@@ -235,6 +235,80 @@ ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMfncZuH
 cZrYRyy9+X68fylEWBMzyg+nhjWkxJlJ2M2LAGY=
 ----
 
+===== Management Operation
+
+Using a management operation the following command will generate and store a new key pair credential under the alias `example` using the ecdsa algorithm with 256 key size:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /subsystem=elytron/credential-store=mycredstore:generate-key-pair(alias=example, algorithm=EC, size=256)
+{"outcome" => "success"}
+----
+
+NOTE: Algorithm and size parameters are optional parameters and if not specified default values will be used.
+
+Key pair credentials can also be imported by specifying a private key in OpenSSH format:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /subsystem=elytron/credential-store=mycredstore:import-key-pair(alias=example, openssh-private-key="-----BEGIN OPENSSH PRIVATE KEY-----
+                                                   b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jdHIAAAAGYmNyeXB0AAAAGAAAABCdRswttV
+                                                   UNQ6nKb6ojozTGAAAAEAAAAAEAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlz
+                                                   dHAyNTYAAABBBAKxnsRT7n6qJLKoD3mFfAvcH5ZFUyTzJVW8t60pNgNaXO4q5S4qL9yCCZ
+                                                   cKyg6QtVgRuVxkUSseuR3fiubyTnkAAADQq3vrkvuSfm4n345STr/i/29FZEFUd0qD++B2
+                                                   ZoWGPKU/xzvxH7S2GxREb5oXcIYO889jY6mdZT8LZm6ZZig3rqoEAqdPyllHmEadb7hY+y
+                                                   jwcQ4Wr1ekGgVwNHCNu2in3cYXxbrYGMHc33WmdNrbGRDUzK+EEUM2cwUiM7Pkrw5s88Ff
+                                                   IWI0V+567Ob9LxxIUO/QvSbKMJGbMM4jZ1V9V2Ti/GziGJ107CBudZr/7wNwxIK86BBAEg
+                                                   hfnrhYBIaOLrtP8R+96i8iu4iZAvcIbQ==
+                                                   -----END OPENSSH PRIVATE KEY-----", passphrase=secret)
+{"outcome" => "success"}
+----
+
+NOTE: The parameter passphrase is optional and can be empty if no passphrase was used to encrypt the key.
```

Comment:
Maybe this note should appear below the next code snippet since it applies to both? WDYT?